integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1955,23 +1955,14 @@ msgstr ""
 msgid "The end of the event can't be before the start of the event"
 msgstr "Das Ende der Veranstaltung kann nicht vor ihrem Beginn sein"
 
-#: cms/forms/events/event_form.py
-msgid ""
-"The maximum duration for events is {} days. Consider using recurring events "
-"if the event is not continuous."
-msgstr ""
-"Die maximale Veranstaltungsdauer beträgt {} Tage. Verwenden Sie "
-"wiederkehrende Veranstaltungen, wenn die Veranstaltung nicht kontinuierlich "
-"stattfindet."
-
-#: cms/forms/events/recurrence_rule_form.py
-msgid "Recurrence ends"
-msgstr "Wiederholung endet"
-
 #: cms/forms/events/recurrence_rule_form.py
 msgid "No recurrence frequency selected"
 msgstr "Keine Häufigkeit der Wiederholung angegeben"
 
+#: cms/forms/events/recurrence_rule_form.py
+msgid "Recurrence DAILY is not allowed anymore"
```

Comment:
```suggestion
msgid "Recurrence daily is not allowed anymore"
```
Or maybe this is better in quotes? Like Recurrence "daily" is not allowed anymore